types/stripe-js/index.d.ts

```diff
@@ -40,6 +40,23 @@ declare module '@stripe/stripe-js' {
     /// https://stripe.com/docs/js/payment_intents
     /////////////////////////////
 
+    /**
+     * Use `stripe.confirmAcssDebitPayment` in the [Accept a payment](/docs/payments/accept-a-payment) flow with [Canadian pre-authorized debit](/docs/payments/acss-debit) payment method when the customer submits your payment form..
+     * When called, it will automatically pop up a modal to collect bank account details and verification, accept the mandate, and confirm the [PaymentIntent](/docs/api/payment_intents) when the user submits the form.
```

Comment:
This should be an absolute URL too. I won't leave a comment on every one of these, but could you do a pass through all the additions and make sure the URLs are absolute?